concent_api/api_testing_common.py

```diff
@@ -66,10 +67,14 @@ def print_fails(cls):
         print(f'Total failed tests : {cls.get_fails()} out of {cls.number_of_run_tests}')
 
 
-def assert_condition(actual, expected, error_message = None):
+def assert_condition(actual, expected, error_message = None, compare_lists_regardless_of_order: bool = False):
```

Comment:
I think that it would be better to create a separate function for comparing lists.

Reply:
as above - created separate function with this check